internal/servers/private_network_classes_server.go

```
@@ -189,6 +267,53 @@ func (s *PrivateNetworkClassesServer) Signal(ctx context.Context,
 	return
 }
 
+func (s *PrivateNetworkClassesServer) findDefaultNetworkClass(ctx context.Context) (*privatev1.NetworkClass, error) {
```

golangci-lint (2.11.4)

[error] func (*PrivateNetworkClassesServer).findDefaultNetworkClass is unused

(unused)